broadlink/cover.py

```diff
@@ -58,3 +59,55 @@ def set_percentage_and_wait(self, new_percentage: int) -> None:
                 time.sleep(0.2)
                 current = self.get_percentage()
         self.stop()
+
+class dooya_new(device):
+    """Controls a Dooya curtain motor."""
+
+    def __init__(self, *args, **kwargs) -> None:
+        """Initialize the controller."""
+        device.__init__(self, *args, **kwargs)
+        self.type = "Dooya DT360E(4F6E)"
+
+    def _send(self, magic1: int, magic2: int, magic3:int, magic4:int) -> int:
```

Comment:
Remove the "magic1" and "magic2" parameters.
Rename "magic3" to "command".
Rename "magic4" to "attribute" and set default to 0.